test/Makefile

```diff
@@ -33,11 +33,11 @@ clean:
 all: module examples/complete
```

Comment:
why are we changing this Makefile? (we used it in all our modules)

Reply:
Updated to the latest from example-module